ee/tuf/autoupdate.go

```diff
@@ -425,6 +426,38 @@ func (ta *TufAutoupdater) FlagsChanged(ctx context.Context, flagKeys ...keys.Fla
 		"pinned_launcher_version", ta.knapsack.PinnedLauncherVersion(),
 		"pinned_osqueryd_version", ta.knapsack.PinnedOsquerydVersion(),
 	)
+
+	// Check for AutoupdateInterval changes
```

Comment:
I would move all of this to the top of this function, I think -- otherwise, the early return if `len(binariesToCheckForUpdate) == 0 || time.Now().Before(ta.initialDelayEnd)` could prevent us from getting here.